.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Feature request
+about: Suggest an idea or ask for a new feature
+---
+
+### Motivation
+<!-- Your motivation behind this feature request. Stating any specific business reason will help everyone in the team to understand the driver. -->
+
+### Details
```

Comment:
Maybe add a "supporting data" section like in big report, to add screenshots or mockups of UI